source/local/local.go

```diff
@@ -38,10 +38,21 @@ const Name = "local"
 // LabelFeature of this feature source
 const LabelFeature = "label"
 
+// RawFeature of this feature source
+const RawFeature = "feature"
+
 // ExpiryTimeKey is the key of this feature source indicating
 // when features should be removed.
 const ExpiryTimeKey = "expiry-time"
 
+// NoLabel indicates whether the feature should be included
+// in exposed labels or not.
+const NoLabel = "no-label"
+
+// NoFeature indicates whether the feature should be included
+// in exposed raw features or not.
+const NoFeature = "no-feature"
```

Comment:
suggestion: we could join the directive names under one `const` block to keep them in the same "context". Also, we could use a prefix to denote that they are directives.

```suggestion
const (
    // ExpiryTimeKey is the key of this feature source indicating
    // when features should be removed.
    DirectiveExpiryTimeKey = "expiry-time"
    
    // NoLabel indicates whether the feature should be included
    // in exposed labels or not.
    DirectiveNoLabel = "no-label"
  
    // NoFeature indicates whether the feature should be included
    // in exposed raw features or not.
    DirectiveNoFeature = "no-feature"
)
```

WDYT?